src/NosCore.GameObject/Providers/MapItemProvider/Handlers/DropHandler.cs

```diff
@@ -38,54 +38,53 @@ public class DropEventHandler : IEventHandler<MapItem, Tuple<MapItem, GetPacket>
     {
         public bool Condition(MapItem item)
         {
-            return (item.ItemInstance.Item.ItemType != ItemType.Map) && (item.VNum != 1046);
+            return (item.ItemInstance!.Item!.ItemType != ItemType.Map) && (item.VNum != 1046);
         }
 
-        public Task Execute(RequestData<Tuple<MapItem, GetPacket>> requestData)
+        public async Task Execute(RequestData<Tuple<MapItem, GetPacket>> requestData)
         {
             var amount = requestData.Data.Item1.Amount;
             var inv = requestData.ClientSession.Character.InventoryService.AddItemToPocket(
-                    InventoryItemInstance.Create(requestData.Data.Item1.ItemInstance,
+                    InventoryItemInstance.Create(requestData.Data.Item1.ItemInstance!,
                         requestData.ClientSession.Character.CharacterId))
                 .FirstOrDefault();
 
             if (inv != null)
             {
-                requestData.ClientSession.SendPacket(inv.GeneratePocketChange((PocketType) inv.Type, inv.Slot));
-                requestData.ClientSession.Character.MapInstance.MapItems.TryRemove(requestData.Data.Item1.VisualId,
+                await requestData.ClientSession.SendPacket(inv.GeneratePocketChange((PocketType) inv.Type, inv.Slot));
+                requestData.ClientSession.Character.MapInstance!.MapItems.TryRemove(requestData.Data.Item1.VisualId,
                     out _);
-                requestData.ClientSession.Character.MapInstance.SendPacket(
+                await requestData.ClientSession.Character.MapInstance.SendPacket(
                     requestData.ClientSession.Character.GenerateGet(requestData.Data.Item1.VisualId));
                 if (requestData.Data.Item2.PickerType == VisualType.Npc)
                 {
-                    requestData.ClientSession.SendPacket(
-                        requestData.ClientSession.Character.GenerateIcon(1, inv.ItemInstance.ItemVNum));
+                    await requestData.ClientSession.SendPacket(
+                        requestData.ClientSession.Character.GenerateIcon(1, inv.ItemInstance!.ItemVNum));
                 }
 
-                requestData.ClientSession.SendPacket(requestData.ClientSession.Character.GenerateSay(
-                    $"{Language.Instance.GetMessageFromKey(LanguageKey.ITEM_ACQUIRED, requestData.ClientSession.Account.Language)}: {inv.ItemInstance.Item.Name[requestData.ClientSession.Account.Language]} x {amount}",
+                await requestData.ClientSession.SendPacket(requestData.ClientSession.Character.GenerateSay(
+                    $"{Language.Instance.GetMessageFromKey(LanguageKey.ITEM_ACQUIRED, requestData.ClientSession.Account.Language)}: {inv.ItemInstance!.Item!.Name[requestData.ClientSession.Account.Language]} x {amount}",
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Split this 219 characters long line (which is greater than 200 authorized).](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5262615)